corehq/apps/users/models.py

```diff
@@ -2354,8 +2355,18 @@ def create(cls, domain, username, password, created_by, created_via, email=None,
         web_user.save()
         web_user.log_user_create(domain, created_by, created_via,
                                  by_domain_required_for_log=by_domain_required_for_log)
+        if TABLEAU_USER_SYNCING.enabled(domain):
+            # Inline import avoids a circular import error
+            from corehq.apps.reports.util import add_tableau_user
+            add_tableau_user(domain, username)
```

Comment:
What happens if this call fails? Same question for the `delete_tableau_user` call below.

Reply:
Displaying an error seems plenty good enough, even a vague one like "Please try again in a few minutes." My concern was just that they user wouldn't know if it failed and would be confused if they tried to take an action with that user before the sync task resolved things.